addons/point_of_sale/static/src/js/models.js

```diff
@@ -351,22 +344,25 @@ class PosGlobalState extends PosModel {
     cashierHasPriceControlRights() {
         return !this.config.restrict_price_control || this.get_cashier().role == 'manager';
     }
-    createAutomaticallySavedOrder(json) {
+    _onReactiveOrderUpdated(order) {
+        order.save_to_db();
+    }
+    createReactiveOrder(json) {
         const options = {pos:this};
         if (json) {
             options.json = json;
         }
         let order = Order.create({}, options);
-        const batchedSaveToDb = batched(() => {
-            order.save_to_db();
+        const batchedCallback = batched(() => {
+            this._onReactiveOrderUpdated(order)
         });
-        order = reactive(order, batchedSaveToDb);
+        order = reactive(order, batchedCallback);
         order.save_to_db();
```

Comment:
Nah. When creating a reactive order, we don't only want to save it to the database, we don't want to override it in the pos_restaurant and to always add it to the updatedOrderSet, otherwise when we'll fetch the orders of a table, we'll keep sending everything to the back end again because we're recreating the orders.

Reply:
`save_to_db` is not saving to the backend. It's poorly named because it's just saving to localStorage (db.js).

Reply:
Yeah but by calling `_onReactiveOrderUpdated` here, we'd run this code
```js
   //@override
    _onReactiveOrderUpdated(order) {
        super._onReactiveOrderUpdated(...arguments)
        if (this.config.iface_floorplan) {
            this.ordersToUpdateSet.add(order);
        }
    }
```
which will ends up saving everything to the database, even the orders you did not modify

Reply:
I'll test this.

Reply:
That's simply the same as calling `this.ordersToUpdateSet.add(order)` right after `order.save_to_db()`, and then when you'll leave the table after fetching the orders, it'll send them